cves/kernel/CVE-2020-28974.yml

```diff
@@ -452,7 +473,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    It may be design mistakes related to handling. The copy feature was only used by only 1 user, 
```

Comment:
- "The copy feature was only used by only 1 user" remove the second only, as it was already used at the beginning of a sentence.
- I would reword the "although they tried to did some tests" part of the sentence, as it is difficult to understand what the sentence is trying to say.